docs/ops/start.md

```diff
@@ -0,0 +1,54 @@
+# Managing StashCache and associated services
+
+Ensure that your `/stash` disk is mounted, and then start `xrootd` and `condor` service.
+
+## Non-authenticated Cache server services
+| **Software** | **Service name** | **Notes** |
+|--------------|------------------|-----------|
+| XRootD | `xrootd@stashcache-cache-server.service` | RHEL7 |
+| HTCondor | `condor.service` | RHEL7  |
+| Fetch CRL | `fetch-crl-cron` | RHEL7 |
+
+## Authenticated Cache server services
+| **Software** | **Service name** | **Notes** |
+|--------------|------------------|-----------|
+| XRootD | `xrootd@stashcache-cache-server-auth.service` | RHEL7 |
+|  | `xrootd-renew-proxy.service` | RHEL7 |
+|  | `xrootd-renew-proxy.timer` | RHEL7  |
+| HTCondor | `condor.service` | RHEL7  |
+| Fetch CRL | `fetch-crl-cron` | RHEL7 |
+
+### Validating Cache server
+To verify that your cache is being monitored properly, run the following command:
```

Comment:
There has to be more of a test than just this.  You should include a `wget` command to test the CVMFS accessibility.

    curl -O http://cache_host:8000/user/dweitzel/public/blast/queries/query1

Or something like that.  I'm sure there is a curl command that doesn't produce output.